Bottom: cf72613e6d1dc7eb290f8e8f8639bae168f675c5
Top:    bfdc883e680b7b29f151270f2a8dc7133b0998a5
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:11:28 +0000

Re-enable soldermask emission


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index efe337c..8d81a8e 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1040,7 +1040,7 @@ object3d_from_soldermask_within_area (POLYAREA *area, int side)
   LayerType *layer;
   POLYAREA *pa;
 
-  return NULL;
+//  return NULL;
 
   poly_Copy0 (&info.poly, area);
   info.side = side;
